.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
 
 install:
   - "pip install tox"
+  - "sudo -H pip install youtube-dl"
```

Comment:
Why would you execute this as sudo?

Reply:
Perhaps it is because the script only needs youtube-dl as a dependency of pafy, and doesn't import youtube-dl directly?

Reply:
@natumbri hmm could me let met check this :smile:

Reply:
Thanks! Sorry for all the commits that keep coming through - I haven't really gotten the hang of git yet, but I'm working it out, hopefully...

Reply:
I think "install:" was the wrong place for youtube-dl, which is why teh sudo was necessary. 
 This works: https://github.com/mopidy/mopidy-youtube/commit/d25e57529eb56bb7f58ca59e87b9807e48fb353b